01-Git_Bash_Make/script.sh

```diff
@@ -13,3 +13,14 @@ fi
 cd /tmp/
 tar -czvf guesanumber.tar.gz ./guesanumber
 
+if [ ! -d /tmp/guesanumber/realese ]
+    then 
+    mkdir /tmp/guesanumber/realese
+elif [ ! -z "$(ls -A /tmp/guesanumber/realese)" ]
```

Comment:
> > But deleting happens only in case if this folder already exists and it's not empty.
> > Shouldn't I check this dir for old versions?
> 
> What is reason to delete empty dir?

It deletes only not empty dir.